### PR TITLE
sl/symheap: simplify the interface for anon stack objects

### PR DESCRIPTION
**Pull request by kdudka**
... and simplify its implementation inside SymHeapCore

### CODE REVIEW

**Comment by versokova** [comment]
Thanks. It looks great.

File: sl/symgc.cc [hunk 1]
```diff
@@ -108,13 +108,14 @@ bool gcCore(SymHeap &sh, TObjId obj, TObjSet *leakObjs, bool sharedOnly)
 
         if (sh.isAnonStackObj(obj))
```
Comment (versokova) [inline sl/symgc.cc:109]: Just a small suggestion: `if (!sh.isAnonStackObj(obj))` because an empty statement looks weird. But it may be as it is now.

Comment (kdudka) [reply sl/symgc.cc]: Good point.  Fixed with 174b0af3450dfbcdddbaa7aeba0996e7109daea5.

**Comment by kdudka** [comment]
Thanks for review!  Merging...

**Pull request closed without merging**